api/renter.go

```diff
@@ -275,8 +287,24 @@ func (api *API) renterContractsHandler(w http.ResponseWriter, _ *http.Request, _
 
 // renterDownloadsHandler handles the API call to request the download queue.
 func (api *API) renterDownloadsHandler(w http.ResponseWriter, _ *http.Request, _ httprouter.Params) {
+	dlq := api.renter.DownloadQueue()
+
+	// Translate []modules.DownloadInfo to []api.DownloadInfo.
+	downloads := make([]DownloadInfo, len(dlq))
+	for i := range dlq {
+		d := dlq[len(dlq)-i-1]
```

Comment:
Does dlq have a promise on what order things will be returned in? Seems like the type of thing that might get optimized and jumbled as we continue to pursue performance (this stuff is likely to end up in a bunch of maps and random-access disk locations)